portfolio/src/main/webapp/README.md

```diff
@@ -0,0 +1,60 @@
+[![Build Status](https://travis-ci.org/BlackrockDigital/startbootstrap-heroic-features.svg?branch=master)](https://travis-ci.org/BlackrockDigital/startbootstrap-heroic-features)
+[![dependencies Status](https://david-dm.org/BlackrockDigital/startbootstrap-heroic-features/status.svg)](https://david-dm.org/BlackrockDigital/startbootstrap-heroic-features)
+[![devDependencies Status](https://david-dm.org/BlackrockDigital/startbootstrap-heroic-features/dev-status.svg)](https://david-dm.org/BlackrockDigital/startbootstrap-heroic-features?type=dev)
+
+## Download and Installation
+
+To begin using this template, choose one of the following options to get started:
+* [Download the latest release on Start Bootstrap](https://startbootstrap.com/template-overviews/heroic-features/)
+* Install via npm: `npm i startbootstrap-heroic-features`
+* Clone the repo: `git clone https://github.com/BlackrockDigital/startbootstrap-heroic-features.git`
+* [Fork, Clone, or Download on GitHub](https://github.com/BlackrockDigital/startbootstrap-heroic-features)
+
+## Usage
+
+### Basic Usage
+
+After downloading, simply edit the HTML and CSS files included with the template in your favorite text editor to make changes. These are the only files you need to worry about, you can ignore everything else! To preview the changes you make to the code, you can open the `index.html` file in your web browser.
+
+### Advanced Usage
+
+After installation, run `npm install` and then run `npm start` which will open up a preview of the template in your default browser, watch for changes to core template files, and live reload the browser when changes are saved. You can view the `gulpfile.js` to see which tasks are included with the dev environment.
+
+You must have npm and Gulp installed globally on your machine in order to use these features.
+
+## Bugs and Issues
+
+Have a bug or an issue with this template? [Open a new issue](https://github.com/BlackrockDigital/startbootstrap-heroic-features/issues) here on GitHub or leave a comment on the [template overview page at Start Bootstrap](http://startbootstrap.com/template-overviews/heroic-features/).
+
+## About
+
+Start Bootstrap is an open source library of free Bootstrap templates and themes. All of the free templates and themes on Start Bootstrap are released under the MIT license, which means you can use them for any purpose, even for commercial projects.
+
+* https://startbootstrap.com
+* https://twitter.com/SBootstrap
+
+Start Bootstrap was created by and is maintained by **[David Miller](http://davidmiller.io/)**, Owner of [Blackrock Digital](http://blackrockdigital.io/).
+
+* http://davidmiller.io
+* https://twitter.com/davidmillerskt
+* https://github.com/davidtmiller
+
+Start Bootstrap is based on the [Bootstrap](http://getbootstrap.com/) framework created by [Mark Otto](https://twitter.com/mdo) and [Jacob Thorton](https://twitter.com/fat).
```

Comment:
Should we get rid of this file?

Looks like a bootstrap README?

Reply:
Yes, I'll delete this